front/components/ConnectorPermissionsTree.tsx

```diff
@@ -46,6 +52,8 @@ function PermissionTreeChildren({
   permissionFilter,
   canUpdatePermissions,
   onPermissionUpdate,
+  depth,
```

Comment:
I can't find where is `depth` used.

Reply:
Good catch I can remove it now, i used it in another implementation to pad-left but now unused :+1: